packages/web-react/src/components/UNSTABLE_Slider/README.md

```diff
@@ -0,0 +1,69 @@
+| `label`          | `string`                 | -       | ✓        | Label text                                                |
+| `max`            | `number`                 | 100     | ✕        | Max value of slider input                                 |
+| `min`            | `number`                 | 0       | ✕        | Min value of slider input                                 |
+| `onChange`       | `() => void`             | -       | ✓        | On input change callback                                  |
+| `step`           | `number`                 | 1       | ✕        | Sets the stepping interval                                |
+| `validationText` | [`string` \| `string[]`] | -       | ✕        | Validation text                                           |
+| `value`          | `number`                 | 0       | ✓        | Input value                                               |
+
+The components accept [additional attributes][readme-additional-attributes].
+If you need more control over the styling of a component, you can use [style props][readme-style-props]
+and [escape hatches][readme-escape-hatches].
+
+# UNSTABLE UncontrolledSlider
+
+## Basic usage
+
+```jsx
+import UNSTABLE_UncontrolledSlider from '../UNSTABLE_UncontrolledSlider';
+
+<UNSTABLE_UncontrolledSlider id="slider-uncontrolled" label="UncontrolledSlider" />;
+```
+
+### API
+
+| Attribute        | Type         | Default | Required | Description                                               |
+| ---------------- | ------------ | ------- | -------- | --------------------------------------------------------- |
+| `helperText`     | `string`     | -       | ✕        | Custom helper text                                        |
+| `id`             | `string`     | -       | ✓        | Input and label identification                            |
+| `isFluid`        | `bool`       | `false` | ✕        | Whether the element spans to the full width of its parent |
+| `isSelected`     | `bool`       | `false` | ✕        | Whether is the item selected                              |
+| `label`          | `string`     | -       | ✓        | Label text                                                |
+| `max`            | `number`     | 100     | ✕        | Max value of slider input                                 |
+| `min`            | `number`     | 0       | ✕        | Min value of slider input                                 |
+| `onChange`       | `() => void` | -       | ✓        | On input change callback                                  |
+| `step`           | `number`     | 1       | ✕        | Sets the stepping interval                                |
+| `validationText` | `string`     | -       | ✕        | Validation text                                           |
+| `value`          | `number`     | 0       | ✕        | Input value                                               |
```

Comment:
here the value is not required?

Reply:
Here it is not necessary, because the state of the uncontrolled value is managed by our hook. In a controlled component, the value must be provided by the developer.